### PR TITLE
Add capture-to-file sample

### PR DESCRIPTION
**Pull request by jimbankoski**
Shows capture from camera, compress and store to a webm file.

### CODE REVIEW

File: samples/capture-to-file/capture-to-file.html [hunk 1]
```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="origin-trial" content="AkLPETKVnJC/njTvpbWOu+cyUhX9kjAvSZOOHgrfmlgn7CGKsW0+pVCbEScZU7y90ogjl93LQ4vH4xE2u5t65wgAAAB3eyJvcmlnaW4iOiJodHRwczovL3dlYmNvZGVjcy1ibG9ncG9zdC1kZW1vLmdsaXRjaC5tZTo0NDMiLCJmZWF0dXJlIjoiV2ViQ29kZWNzIiwiZXhwaXJ5IjoxNjE1MjMzMTk4LCJpc1N1YmRvbWFpbiI6dHJ1ZX0=">
```
Comment (chcunningham) [inline samples/capture-to-file/capture-to-file.html:6]: Nit: can you use the OT token 
`AlfvFCF6XVoauEvWVCLsov9cuGlQbz6aTIO6ZdLWHG5YmZpdqON+7lXo7ZUHK+IkkdkJji+7Fxnr/yTu7Iu5Og0AAABjeyJvcmlnaW4iOiJodHRwczovL3czYy5naXRodWIuaW86NDQzIiwiZmVhdHVyZSI6IldlYkNvZGVjcyIsImV4cGlyeSI6MTYyNjIyMDc5OSwiaXNTdWJkb21haW4iOnRydWV9`

That's the one [Thomas setup](https://github.com/w3c/webcodecs/commit/11cab73a899c3452a1e1237c1213633372f1b391) for the other files.

**Review by chcunningham — CHANGES_REQUESTED** [review]
For folks reading along, Jim and I spoke offline. Fixes pending for 2 bugs:

1) seeking busted: needs one line fix to plumb keyframe flag through 
2) additional files (after the first) appear to not start w/ key frame. Not sure yet of the cause.

File: samples/capture-to-file/capture-to-file.html (hunk 1 above)
Comment (jimbankoski) [reply samples/capture-to-file/capture-to-file.html]: Okay I think both problems fixed closing the encoder in between runs  seems to be the fix ( I think).  

OT fixed as is the keyframe flagging.    Still might be best described as a toy muxer.   It doesn't even handle audio yet.  :)

**Review by chcunningham — APPROVED** [review]
LGTM! Thanks a ton @jimbankoski

**Pull request merged**